packages/teleport/src/Databases/ConnectDatabase/ConnectDatabase.story.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import ConnectDatabase from './ConnectDatabase';
```

Comment:
Starting to think we should rename these to `ConnectDialogue` and `AddDialogue`, bc the root folder is `Databases` and it makes it clearer what the component is, what do you think?

Reply:
I agree, `ConnectDatabase` `AddDatabase` might not be descriptive enough

Reply:
I think `ConnectDialog` is more consistent with the way we spell `Dialog` in other places in the code than `Dialogue`, eg. `Dialog`, `DialogTitle`, `DialogContent`, `dialogCss`